src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2366,6 +2371,55 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
             return null;
         }
 
+        private BoundNode VisitRefConditionalOperator(BoundConditionalOperator node)
```

Comment:
It looks like `VisitRefConditionalOperator` has the same structure as `VisitConditionalOperator`. If so, consider handling both cases in `VisitConditionalOperator` with a `VisitConditionalOperand(BoundExpression operand, RefKind refKind)` helper to handle each operand.